docs/concepts.md

```diff
@@ -1,37 +1,37 @@
 Concepts
 ========
 
-Simplistic Overview of request processing
+Overview of request processing
 -----------------------------------------
 
-A request is received on the edge server (fastly) and is associated to a service configuration.
-The configuration contains a varnish cache script (VCL) that extracts processing information from
-the request. those are:
+An HTTP request is received on the edge server (fastly) and is associated to a [service configuration][fastly-services].
+The configuration contains a varnish cache script ([VCL][vcl]) that extracts processing information from
+the request. Those are:
 
 - requested resource
 - selector and extension
 - strain name
```

Comment:
We should define what a strain is, and link to that here.

Reply:
I think the best (and only?) description we currently have is this:
https://github.com/adobe/helix-shared/blob/master/CONFIGURATION.md

Reply:
While this describes, sort of, the few properties strains contain (but it doesn't seem like an exhaustive list!), it doesn't set out a concise definition. That is what I'm looking for here. Any suggestions? Here is my attempt:

> A _strain_ is a configuration that maps HTTP request parameters to a content repository that is processed by one or more helix pipelines. Taken together, a helix configuration contains a set of strains that describe what content (optionally processed by what pipelines) is returned as responses to HTTP requests.

To me the above sounds sort of like a "router" in other kinds of web development.

Just random thoughts, but figured better to offer a suggestion than to simply point out the problem 😛

Reply:
Thanks for the proposal! I propose 2 minor changes and have 1 question:
- strain selection is not just HTTP request params, but also (even mainly?) URL based
- it should be mentioned that a strain is a combo of code, content and static repos
- should we mention GitHub, Fastly and Runtime, or abstract them away?
So something like:
> A _strain_ is the part of a Helix configuration that maps HTTP requests to a combination of GitHub repositories for code, content and static files. In the simplest case, a single repository can be used for all three. 
A Helix configuration consists of one or multiple strains, which result in the deployment of matching OpenWhisk actions to Adobe I/O Runtime, and instruct Fastly which of them to invoke in response to HTTP requests.

wdyt?

Reply:
I've added this to the PR, but I do think it is useful to somehow mention pipeline, no? A strain defines the main (or only parts of?) a helix pipeline, right?

Reply:
Hmm, not really - I think a strain primarily defines where code and content should be taken from. Which pipeline gets triggered for which request is actually defined by the code, or the extensions being used... 

but I'll happily stand corrected corrected 😛